package.json

```diff
@@ -59,6 +59,7 @@
     "grunt-csscomb": "3.0.0",
     "grunt-git-authors": "2.0.0",
     "grunt-html": "1.6.0",
+    "grunt-htmllint": "^0.2.7",
```

Comment:
We use firm versions to do our best to avoid weird issues with different people having different versions.